qiskit/_openquantumcompiler.py

```diff
@@ -77,10 +78,20 @@ def compile(qasm_circuit, basis_gates='u1,u2,u3,cx,id', coupling_map=None,
         if not silent:
             print("post-mapping properties: %s"
                   % compiled_dag_circuit.property_summary())
+    # choose output format
+    if format is 'dag':
```

Comment:
Nitpick: would it be possible to change the comparison to equality testing (`format == 'dag'`) instead of identity testing (`format is 'dag'`)? It is probably not a strong distinction in this particular usage, but in general, when comparing _strings_  we usually want to compare that "_the contents are equal_" instead of "_the object is the same_":

```python
>>> a = 'hello'
>>> b = ''.join('hello')
>>> a is 'hello'
True
>>> a == 'hello'
True
>>> b is 'hello'
False
>>> b == 'hello'
True
>>> id(a)
140454905654776
>>> id('hello')
140454905654776
>>> id(b)
140454905655000
```

Hopefully in the future this kind of nitpicks will be easily picked up from the linter output (in this case it emits a `Comparison to literal (literal-comparison)`) instead of me being a bit annoying!

Reply:
sounds good